backend/app/adapter/db/url_test.go

```diff
@@ -186,6 +188,92 @@ func TestURLSql_Create(t *testing.T) {
 	}
 }
 
+func TestSearchURLSql_GetURLByVisibility(t *testing.T) {
+	var searchResponse search.URLSearchResponse
```

Comment:
Search logic should not located inside repository test

Reply:
Moved to `userurl_test.go`